examples/echo.rs

```diff
@@ -163,6 +158,15 @@ fn main() {
             }
         });
 
+    // handle incoming friend connections by just accepting all of of them
+    let friend_connection_c_2 = friend_connections.clone();
```

Comment:
In rust it's ok to have several `let`s with the same name, so we can call it just `friend_connections_c`. Also let it be plural to be consistent with other names.